File: C++/2390. Removing Stars From a String.cpp
class Solution {
public:
    string removeStars(string s) {
        vector<char>a;
        for(char c:s){
            if(c=='*'){
                if(a.size()!=0)
                a.pop_back();
            }
            else
            a.push_back(c);

        }
        string b="";
        for(char c:a){
            b+=c;
        }
        return b;
    }
};

File: JAVA/2390. Removing Stars From a String.java
class Solution {
    public String removeStars(String s) {
        List<Character> a=new ArrayList<>();
        for(Character c:s.toCharArray()){
            if(c=='*'){
                if(a.size()!=0)
                a.remove(a.size()-1);
            }
            else
            a.add(c);
        }
        String b="";
        for(Character c:a){
            b+=c;
        }
        return b;
    }
}